src/Sylius/Bundle/LocaleBundle/composer.json

```diff
@@ -56,6 +57,9 @@
     "extra": {
         "branch-alias": {
             "dev-main": "2.0-dev"
+        },
+        "symfony": {
+            "require": "7.0.*"
```

Comment:
that'd be better to require "7.1.*", but we require this version on other packages.